### ISSUE TITLE
Screenreader text for application block uses incorrect page number

### ISSUE THREAD

**Issue by jeanniefu** [seattle-uat/civiform#2619]
Feedback from a11y experts during an a11y audit.

Severity (Critical/Serious/Moderate/Good to have): Moderate

Screen reader text for application block uses incorrect page number. In the below example, the h1 heading uses "1 of 7" but the UI shows "2 of 7". The screen reader text should be updated to "2 of 7".
<img width="1466" alt="Screen Shot 2022-06-01 at 5 21 16 PM" src="https://user-images.githubusercontent.com/9094240/171522576-c07bd441-3adb-4589-ae2f-ecf35a591c7e.png">
